src/libraries/System.Private.CoreLib/src/System/Reflection/AssemblyNameParser.cs

```diff
@@ -51,7 +60,7 @@ public static RuntimeAssemblyName Parse(string s)
             byte[]? pkt = null;
             AssemblyNameFlags flags = 0;
 
-            LowLevelList<string> alreadySeen = new LowLevelList<string>();
+            List<string> alreadySeen = new List<string>();
```

Comment:
Should this be ifdef`d to be `LowLevelList` in CoreRT or `List` is ok?

Reply:
If the NativeAOT tests pass, it is ok to change it to List.

Reply:
I think it was passing with this in previous iterations.
Everything was passing except Mono LLVM, that I did not wait for to complete, since it takes so long.